# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qt/stddef.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/tools/qvaluelist.h	2008-01-15 20:09:13.000000000 +0100
+++ ./src/tools/qvaluelist.h	2013-02-21 16:39:02.268736404 +0100
@@ -45,6 +45,7 @@
 #include "qtl.h"
 #include "qshared.h"
 #include "qdatastream.h"
+#include <stddef.h>
 #endif // QT_H
 
 #ifndef QT_NO_STL
--- ./src/tools/qmap.h	2008-01-15 20:09:13.000000000 +0100
+++ ./src/tools/qmap.h	2013-02-21 16:39:23.858881970 +0100
@@ -47,6 +47,7 @@
 #include "qdatastream.h"
 #include "qpair.h"
 #include "qvaluelist.h"
+#include <stddef.h>
 #endif // QT_H
 
 #ifndef QT_NO_STL

